pkg/registry/apps/deployment/storage/storage.go

```diff
@@ -383,9 +384,17 @@ func (i *scaleUpdatedObjectInfo) UpdatedObject(ctx context.Context, oldObj runti
 		return nil, errors.NewNotFound(apps.Resource("deployments/scale"), i.name)
 	}
 
+	defaultGroupVersion := schema.GroupVersion{Group: "apps", Version: "v1"}
+	if requestInfo, found := genericapirequest.RequestInfoFrom(ctx); found {
```

Comment:
That's great, I'm just curious how we would end-up with a `requestInfo` that doesn't know about a version we're serving here. Also I think we should rename `defaultGroupVersion` to merely `groupVersion` since default is a little misleading.

Reply:
I have renamed the variable and cleaned up the commit history.